docs/en/guide/gateway.md

```diff
@@ -495,13 +495,13 @@ USAGE:
   mc admin policy COMMAND [COMMAND FLAGS | -h] [ARGUMENTS...]
 
 COMMANDS:
-  add     add new policy
-  remove  remove policy
-  list    list all policies
-  info    show info on a policy
-  set     set IAM policy on a user or group
-  unset   unset an IAM policy for a user or group
-  update  Attach new IAM policy to a user or group
+  add     Add a new policy
+  remove  Remove a policy
+  list    List all policies
+  info    Show info on a policy
+  set     Set IAM policy on a user or group
+  unset   Unset an IAM policy for a user or group
+  update  Attach a new IAM policy to a user or group
```

Comment:
this content is extracted directly from minio client output, do not modify @CaitinChen